scripts/cmake/DocumentationSetup.cmake

```diff
@@ -38,4 +38,36 @@ if(DOXYGEN_FOUND)
 
     configure_file(Documentation/Doxyfile.in ${PROJECT_BINARY_DIR}/Doxyfile)
 
+    if (NOT BASH_TOOL_PATH STREQUAL "BASH_TOOL_PATH-NOTFOUND"
+            AND NOT PYTHON_EXECUTABLE STREQUAL "PYTHON_EXECUTABLE-NOTFOUND")
+        set(doc_use_external_tools TRUE)
+    else()
+        set(doc_use_external_tools FALSE)
```

Comment:
I guess the setting of the variable should be the other way around? 

Also if this var is just used once I would put the tools check into the if-command below and remove the variable.